monitoring-custom-metrics/deployment/service-monitor.yaml

```diff
@@ -16,5 +16,11 @@ spec:
   endpoints:
```

Comment:
The targetLabels section above should not be required, please double-check and remove

Reply:
Not required